simulator/src/main/resources/reference.conf

```diff
@@ -452,6 +452,11 @@ caffeine.simulator {
     queues = 256
   }
 
+  camp {
+  # Parameter precision p
+  p = 5
```

Comment:
Please call the parameter `precision` and the Java code to use that variable name instead of the non-descriptive `p`.

Reply:
Changed it to `precision` in both reference and in the code itself